pyproject.toml

```diff
@@ -1,17 +1,19 @@
 [tool.poetry]
-name = "arcaflow-plugin-template-python"
+name = "arcaflow-plugin-horreum-client"
 version = "0.1.0"
-description = ""
+description = "arcaflow plugin for horreum client"
 authors = ["Arcalot"]
 license = "Apache-2.0+GPL-2.0-only"
 
 packages = [
-   { include="arcaflow_plugin_template_python.py", from="./arcaflow_plugin_template_python"  },
+   { include="horreum_client_plugin.py", from="./arcaflow_plugin_horreum_client"  },
 ]
 
 [tool.poetry.dependencies]
 python = "^3.9"
 arcaflow-plugin-sdk = "^0.13.0"
+requests = "^2.31.0"
+responses = "^0.24.1"
```

Comment:
Keep in mind that using the ^ notation is fairly loose, which could cause problems when using a later build of the same source, like when you use the Python client.